client/consensus/aura/src/import_queue.rs

```diff
@@ -98,9 +87,14 @@ where
 			}
 
 			Ok(CheckedHeader::Checked(header, (slot, seal)))
-		} else {
-			Err(Error::BadSignature(hash))
-		}
+		},
+		Err(SealVerificationError::Deferred(header, slot)) =>
```

Comment:
Nit. Maybe we can add a `From<SealVerificationError>` implementation for aura `Error` or keep it simple by returning directly an Aura `Error` from `check_header_slot_and_seal`?

Reply:
IMO it's bad practice to have these huge overarching error types and prefer to keep it localized. For that reason I'd prefer not to return an Aura `Error`. Adding a `From<SealVerificationError>` is stylistic, but as currently exists, the code would have this match statement in one place.